Makefile

```
@@ -24,9 +24,15 @@ clean:
 	$(MAKE) -C guides/ clean
 	rm -rf $(DEST) web/output/
 
-html: prep
+html: build-foreman-el build-foreman-deb build-katello
+
+build-foreman-el: prep
 	$(MAKE) -C guides/ html BUILD=foreman-el
+
+build-foreman-deb: prep
 	$(MAKE) -C guides/ html BUILD=foreman-deb
+
+build-katello: prep
 	$(MAKE) -C guides/ html BUILD=katello
```

```suggestion
build-%: prep
	$(MAKE) -C guides/ html BUILD=$*
```

Makes it easier to create new targets?

No idea how to make the same for `.PHONY`.

---

Ah, you could make them all depend on a `FORCE` target and make that `PHONY`. works in https://github.com/theforeman/foreman-ansible-modules/blob/develop/Makefile